component-samples/demo/manufacturer/Dockerfile

```diff
@@ -17,7 +17,7 @@ WORKDIR /home/manufacturer/
 COPY ./lib ./lib/
 COPY ./certs ./certs/
 COPY manufacturer.jar .
-COPY manufacturer_keystore.p12 .
+COPY manufacturer_keystore.p12 owner.pem* reseller.pem* .
```

Comment:
This line should be changed as below.

COPY manufacturer_keystore.p12 owner_pub_keys.pem reseller_pub_keys.pem ./

Reply:
Done.